openreview/venue/webfield/recommendationWebfield.js

```diff
@@ -0,0 +1,48 @@
+// Webfield component
+const reviewerGroup = domain.content.reviewers_id?.value
+const startParam = `${domain.content.area_chairs_assignment_id?.value},tail:${user.profile.id}`
+const traverseParam = `${entity.id}`
+const browseInvitations = []
+if (domain.content.reviewers_affinity_score_id?.value) {
+  browseInvitations.push(domain.content.reviewers_affinity_score_id?.value)
+}
+
+if (domain.content.bid_name?.value) {
+  browseInvitations.push(`${reviewerGroup}/-/${domain.content.bid_name?.value}`)
+}
+
+const otherParams = `&hide=${domain.content.reviewers_conflict_id?.value}&maxColumns=2&version=2&referrer=[Recommendation Instructions](/invitation?id=${entity.id})`
+
+return {
+  component: 'CustomContent',
+  properties: {
+    HeaderComponent: {
+      component: 'VenueHeader'
+    },
+    header: {
+      title: domain.content.title?.value,
+      subtitle: domain.content.subtitle?.value,
+      website: domain.content.website?.value,
+      contact: domain.content.contact?.value,
+      instructions: '  '
+    },
+    BodyComponent: {
+      component: 'MarkdownContent'
+    },
+    content: `
+<div>
```

Comment:
can this be markdown or it has to be HTML? I would like something simple for the PCs to edit the instructions.

Reply:
I ended up adding these instructions to the instructions field of the header, since I **think** I need to use HTML for the button and the content does not seem to support both HTML and markdown?

Reply:
good idea!